src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/controller/VisitSessionController.kt

```diff
@@ -85,6 +88,63 @@ class VisitSessionController(
     return sessionService.getVisitSessions(prisonCode, prisonerId, min, max)
   }
 
+  @PreAuthorize("hasRole('VISIT_SCHEDULER')")
+  @GetMapping(VISIT_SESSIONS_AVAILABLE_CONTROLLER_PATH)
+  @Operation(
+    summary = "Returns only available (not fully booked / already booked for prisoner) visit sessions for visit restriction and within the reservable time period",
```

Comment:
`(not fully booked / already booked for prisoner)`
not sure you need to put this here